src/scripts/main.js

```diff
@@ -1,3 +1,315 @@
+
+  if (e.target.tagName !== 'TH') {
+    return;
+  }
+
+  sorterByTitleName(th.cellIndex, th.textContent);
+});
+
+document.querySelector('.saveButton').addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const formElements = form.elements;
+
+  for (const input of inputs) {
+    input.addEventListener('blur', () => {
+      input.style.outline = '';
+    });
+
+    if (!inputValidation(input.name, input.value)) {
+      input.style.outline = '1px solid red';
+      input.focus();
+
+      return;
+    }
+  }
+
+  tbody.insertAdjacentHTML('beforeend', `
+    <tr>
+      <td>${capitalizer(formElements.name.value)}</td>
+      <td>${capitalizer(formElements.position.value)}</td>
+      <td>${formElements.office.value}</td>
+      <td>${formElements.age.value}</td>
+      <td>${salaryConverter(formElements.salary.value)}</td>
+    </tr>
+  `);
+
+  pushNotification('Success', 'Employee was added', 'success');
+
+  form.reset();
+});
+
+tbody.addEventListener('dblclick', (e) => {
+  table.style.userSelect = 'none';
+
+  const td = e.target.closest('td');
+
+  if (e.target.tagName !== 'TD') {
+    return;
+  }
+
+  const cellContent = td.textContent;
+  const currentColumnTitle
+    = table.rows[0].cells[td.cellIndex].textContent.toLocaleLowerCase();
+  const input = document.createElement('input');
+
+  input.classList.add('cell-input');
+  input.setAttribute('type', 'text');
+
+  input.value = cellContent;
+
+  if (currentColumnTitle === 'salary') {
+    input.value = salaryConverter(cellContent);
+  }
+
+  td.textContent = '';
+  td.append(input);
+  input.focus();
+
+  input.onblur = () => {
+    if (!input.value || input.value.trim().length === 0) {
+      input.replaceWith(cellContent);
+    }
+
+    if (inputValidation(currentColumnTitle, input.value)
+    ) {
+      if (currentColumnTitle === 'salary') {
+        input.replaceWith(salaryConverter(input.value));
+      }
+
+      input.replaceWith(capitalizer(input.value));
+
+      pushNotification(
+        capitalizer(currentColumnTitle), 'Successfully changed', 'success'
+      );
+    }
+  };
+
+  input.addEventListener('keypress', enterHandler);
+
+  function enterHandler(ev) {
+    if (ev.key === 'Enter' && !input.value) {
+      input.replaceWith(cellContent);
+    }
+
+    if (ev.key === 'Enter'
+      && inputValidation(currentColumnTitle, input.value)
+    ) {
+      if (currentColumnTitle === 'salary') {
+        input.replaceWith(salaryConverter(input.value));
+      }
+
+      input.replaceWith(capitalizer(input.value));
+
+      pushNotification(
+        capitalizer(currentColumnTitle), 'Successfully changed', 'success'
+      );
+    }
+  }
+});
+
+function inputValidation(inputName, inputValue) {
+  if (!inputValue || inputValue.trim().length === 0) {
+    pushNotification(
+      capitalizer(inputName), ' Cannot be empty', 'error'
+    );
+
+    return false;
+  }
+
+  if (inputName === 'name' && inputValue.trim().length < 4
+  ) {
```

Comment:
move expressions to variables

```suggestion
  if (inputName === 'name' && inputValue.trim().length < 4) {
```